coral.py

```diff
@@ -92,6 +93,56 @@ def center_prompt(title, subtitle):
         sys.exit()
 
 
+## Get and save highscore from/in a file
+
+def save_high_score(score):
+    filename = "data/highscore.txt"
```

Comment:
how about moving the `filename` constant to be grouped with the other constants in the `## Game customization` section for better organization and consistency?

Reply:
changed it!